src/components/core/products/CustomTab.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component, useEffect, useState } from "react";
```

Comment:
Remove products folder from **core** folder, as core components are dumb components and these are not dumb components

Reply:
Done .